index.html

```diff
@@ -834,6 +834,20 @@ <h3>
             methods for networked presentation displays:
           </p>
           <dl>
+            <dt>
+              <a href="https://csa-iot.org/" id="csa">Connectivity Standards
+              Alliance</a>
+            </dt>
+            <dd>
+              The Connectivity Standards Alliance (CSA) develops <a href=
+              "https://buildwithmatter.com/">Matter</a>, an industry–unifying
+              standard for reliable and secure connectivity across smart home
+              devices, mobile app and cloud services. The Second Screen Working
+              Group will evaluate features of Matter such as IP-based networking
+              technologies for discovery, authentication, transport and media
+              casting. The Working Group will discuss updates to the Open Screen
```

Comment:
Maybe s/media casting/media streaming/, as casting is sort-of a Google thing.

Reply:
I made the change. FWIW, I had used "casting" because that seems to be the term that Matter uses, see [CHIP TV casting app example](https://github.com/project-chip/connectedhomeip/tree/master/examples/tv-casting-app/linux#chip-tv-casting-app-example) (and I could not find information about possible support for what we would call the 1UA scenario).